src/Aspire.Hosting/ApplicationModel/CustomResourceSnapshot.cs

```diff
@@ -166,20 +170,23 @@ public sealed record UrlSnapshot(string Name, string Url, bool IsInternal);
 /// <param name="Target">The target of the mount.</param>
 /// <param name="MountType">Gets the mount type, such as <see cref="VolumeMountType.Bind"/> or <see cref="VolumeMountType.Volume"/></param>
 /// <param name="IsReadOnly">Whether the volume mount is read-only or not.</param>
+[DebuggerDisplay("{Source}", Name = "{Target}", Type ="{MountType}")]
```

Comment:
```suggestion
[DebuggerDisplay("{Source}", Name = "{Target}")]
```
For everything else, type is a .NET type. Should be consistent.

Is it valid to have multiple volumes with the same target? It doesn't seem like it is, but I'd like to double check.

Reply:
Trying to add duplicate targets results in an error when starting the container

```cs
var cache = builder.AddRedis("cache")
    .WithDataVolume()
    .WithBindMount("s:\\", "/mnt/test")
    .WithBindMount("c:\\temp", "/mnt/test")
    ;
```

![image](https://github.com/user-attachments/assets/acd78ca8-85a6-4afd-92e1-63eb0a630603)